scimusmn/app-template
# Extch 72 update gatsby core

This PR updates the Gatsby version used in `app-template` to [V4](https://www.gatsbyjs.com/gatsby-4/) and addresses [EXTCH-72](https://smm.atlassian.net/browse/EXTCH-72?atlOrigin=eyJpIjoiNWE0ZDQzMTZmNTJhNDNjMGEwMzAzMTI1ZjZhNzc5OGIiLCJwIjoiaiJ9).

This PR also removes these default UI/Style frameworks:
- SASS
- Bootstrap
- Reactstrap

This upgrade allows us to use `Content Sync` for Contentful previews. Steps for doing so are documented [here](https://smm.atlassian.net/wiki/spaces/MED/pages/2159509505/Contentful+driven+applications#Add-Gatsby-Cloud-App-in-Contentful).

To test:

- Pull down this branch.
- Step through initial `app-template` setup. 
- Make sure Gatsby V4 site successfully builds and behaves as expected.
